src/main/resources/l10n/Menu_en.properties

```diff
@@ -56,7 +56,7 @@ Paste=&Paste
 Preferences=&Preferences
 Previous_tab=&Previous_tab
 Quit=&Quit
-Recent_files=&Recent_files
+Recent_databases=Recent_databases
```

Comment:
Please add the `&` sign in front of the 'R' of the translated string as before. In the menu properties this indicates which letter should be used as mnemonic for the underlying action.